src/webapp/src/client/app/common/directives/wall/wall.component.js

```diff
@@ -22,28 +22,28 @@
         '$state',
         '$scope',
         '$window',
-        'wallService'
+        'wallService',
+        'employeeListRepository',
     ];
 
-    function wallController($state, $scope, $window, wallService) {
+    function wallController($state, $scope, $window, wallService, employeeListRepository) {
         /*jshint validthis: true */
         var vm = this;
-
         $window.onscroll = scrollHandler;
         vm.wallServiceData = wallService.wallServiceData;
         vm.getCurrentWallId = wallService.getCurrentWallId;
         vm.stateParams = $state.params;
 
         vm.createPost = createPost;
         vm.reloadWall = reloadWall;
+        vm.getEmployeeList = getEmployeeList;
 
         init();
-
         ////////
 
         function init() {
             wallService.initWall(vm.isWallModule, vm.wallId);
-
+            vm.getEmployeeList();
```

Comment:
This seems wrong, concerns are mixed.. EmployeeList shouldn't be needed when you are loading walls.
I would suggest to look at how users selection is implemented in `/api/Kudos/GetUsersForAutocomplete?s=aaa`